dir.props

```diff
@@ -202,6 +186,25 @@
     <DefineConstants>$(DefineConstants),TRACE</DefineConstants>
   </PropertyGroup>
 
+  <!-- Provide defaults for TestNugetRuntimeId -->
+  <PropertyGroup Condition="'$(TestNugetRuntimeId)'== ''">
+    <TestNugetRuntimeId Condition="'$(OSGroup)'=='Windows_NT'">win7-x64</TestNugetRuntimeId>
+    <TestNugetRuntimeId Condition="'$(OSGroup)'=='Linux'">ubuntu.14.04-x64</TestNugetRuntimeId>
+    <TestNugetRuntimeId Condition="'$(OSGroup)'=='OSX'">osx.10.10-x64</TestNugetRuntimeId>
+    <TestNugetRuntimeId Condition="'$(OSGroup)'=='FreeBSD'">ubuntu.14.04-x64</TestNugetRuntimeId>
```

Comment:
I guess this is just supposed to be a "close enough" default platform, as none of them will really work out-of-the-box on FreeBSD. Although I thought that OSX would be "more compatible" than Ubuntu?